src/main/java/uk/gov/ida/verifyserviceprovider/configuration/EuropeanIdentityConfiguration.java

```diff
@@ -16,13 +16,13 @@
 public class EuropeanIdentityConfiguration extends EidasMetadataConfiguration {
 
     private TrustStoreConfiguration trustStoreConfiguration;
-    private String hubConnectorEntityId;
+    private URI hubConnectorEntityId;
```

Comment:
It doesn't have to be necessarily URI I believe 🤔

Reply:
Hurm, I think in SAML the entityIds are all URIs 🤔

Reply:
Yep, you might be right!

Reply:
OK, higher powers agree with you :)